### ISSUE TITLE
Suggestions don't show up on several apps . Question regarding the clipboard button and please add documentations about OpenBoards features.

### ISSUE THREAD

**Issue by Anonymous2716** [dslul/openboard#763]
When I press and hold the clipboard button it pastes the text of my primary clipboard. But what else does it send along with it? Because whenever I paste something with  the button in termux (https://github.com/termux/termux-app) it immediately runs the command as if I also pressed the entered button.

Also lot of the people Dont know about OpenBoards features please add documentations about it on the github page or somewhere in the app it self.

Also i noticed that OpenBoard doesn't show suggestions in many apps and in webpages but setimes it works right . not always.

**Comment by RHJihan** [comment]
Suggestions not showing everywhere is a known issue. #660
Will be fixed when #749 is merged.

**Comment by Anonymous2716** [comment]
Alright then only my question remains.

**Comment by oblab** [comment]
Hi, i just discover Openboard, the best keyboard i've ever tried on Android, and open-source, great job !
The only big problem is the suggestions for me, especially on Vivaldi browser : in the URL bar, no suggestions, but right  now, while i'm writting here, suggestions work. (As someone mentionned it, it doesn't work in Google play either, but i dont care, it's mainly disabling in the browser)

When do you think it will ne fixed ? Next version ?
Thanks!

**Comment by Anonymous2716** [comment]
> it's mainly disabling in the browser.

That's true. Sometimes I have noticed this on other apps too. Like the launcher application.

**Comment by jnnkB** [comment]
See this comment, which probably belongs to this issue: https://github.com/openboard-team/openboard/pull/749#issuecomment-1507084878
